pallets/xyk-liquidity-mining/Cargo.toml

```diff
@@ -0,0 +1,65 @@
+[package]
+name = "pallet-xyk-liquidity-mining"
+version = "1.0.0"
+description = "Liquidity mining"
+authors = ["GalacticCouncil"]
+edition = "2021"
+homepage = "https://github.com/galacticcouncil/Basilisk-node"
+license = "Apache 2.0"
+repository = "https://github.com/galacticcouncil/Basilisk-node"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.1.5", features = ["derive", "max-encoded-len"], default-features = false }
+scale-info = { version = "2.1.2", default-features = false, features = ["derive"] }
+
+
+sp-arithmetic = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.28", default-features = false }
+log = { version = "0.4.17", default-features = false }
+
+# ORML dependencies
+orml-traits = { git = "https://github.com/open-web3-stack/open-runtime-module-library", branch = "polkadot-v0.9.28", default-features = false }
+
+# HydraDX dependencies
+warehouse-liquidity-mining = { package="pallet-liquidity-mining", git = "https://github.com/galacticcouncil/warehouse", rev="89bde6d2b8bad2263c554eb3d83b76d877971136", default-features = false }
+hydradx-traits-lm = { package="hydradx-traits", git = "https://github.com/galacticcouncil/warehouse", rev="89bde6d2b8bad2263c554eb3d83b76d877971136", default-features = false }
+
+pallet-nft = { git = "https://github.com/galacticcouncil/warehouse", branch="polkadot-v0.9.28", default-features = false }
+hydradx-traits = { git = "https://github.com/galacticcouncil/warehouse", branch="polkadot-v0.9.28", default-features = false }
```

Comment:
yes, it's intentional. Other pallets (xyk, nft) are using traits from master and LM is using trait from LM branch. 
In my opinion warehouse LM should be merged first before we updates all the pallets to use the same trait. Otherwise we would have to do all the traits changes in the warehouse LM branch